omnn/math/Valuable.cpp

```diff
@@ -1801,8 +1801,12 @@ bool Valuable::SerializedStrEqual(const std::string_view& s) const {
     {
         if (exp)
             return exp->operator<(v);
+        // Handle NaN cases first (IEEE 754-2019 section 6.2.1)
+        // "When either operand of a relational operation is NaN, the result is false"
```

Comment:
When quoting anything, double-check that exact quoted line is actually there in the source